.rubocop.yml

```
@@ -2,6 +2,7 @@ require:
   - rubocop-rails
   - rubocop-rspec
   - rubocop-performance
+  - ./rubocop/cops/required_inverse_of_relations.rb
```

As we're using `CustomCops/RequiredInverseOfRelations` for the rule's name, should we use it instead? 🤔 

```suggestion
  - ./rubocop/custom_cops/required_inverse_of_relations.rb
```

---

Good point. It's not a strict requirement to follow the naming convention of Zeitwerk here as we manually requires the file. However, it makes it easier to locate the file. That's a win for sure

fd294bb0350c2dd9e4151390b5efd7533d704a34